src/content/api/media-uploads.mdoc

```diff
@@ -44,3 +44,84 @@ When the Media Upload was updated.
 The User or API Key that updated the Media Upload.
 {% /property %}
 {% /properties %}
+
+{% endpoint method="POST" path="/api/media-uploads" %}
+## Create a presigned URL for Media Upload EXPERIMENTAL
```

Comment:
We might want to check this with Junjie - I think we will want to use badges when the route is experimental 🤠

Reply:
Looks like its still being used by the farmlands connection page - do you think we should replace it with Rows and Columns in this PR?

Reply:
Yeah I agree

Reply:
I've added a task for styling experimental endpoints. I'd say we want a badge for this which is not part of the anchor tag so we don't break anchor tags when they come out of experimental phase